pkg/operator/ceph/object/objectstore.go

```diff
@@ -196,6 +277,26 @@ func decodeID(data string) (string, error) {
 	return id.ID, err
 }
 
+func decodeMasterZoneID(data string) (string, error) {
```

Comment:
I'd suggest just having a single method for `decodeMasterZone()`, then the caller can decide which field it wants to use from the `zoneGroupType` struct.